Priyaa/HTML Basics Remake/Assignment-3.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Assignment-Registration Form</title>
+</head>
+
+<body>
+    <h1><b>Register</b></h1>
+    <form id="registration_form" method="post">
+        <b>First name:</b>
+        <input type="text" name="firstname" placeholder="First Name" autocomplete="given-name" required>
+        <b>Last name:</b>
+        <input type="text" name="lastname" placeholder="Last Name" autocomplete="family-name" required>
+        <br><br>
+
+        <b>Gender:</b>
+        <label for="male">Male</label>
+        <input type="radio" name="gender" id="male" value="male" autocomplete="sex" required>
```

Comment:
```suggestion
        <input type="radio" name="gender" id="male" value="male" required>
```

Radio and checkboxes don't have autocomplete. `sex` autocomplete works for text input.